deployments/kustomize/cronjobs/BACKUP-update-current-year-cron.yaml

```diff
@@ -0,0 +1,216 @@
+# this is a backup of the working config for the update-current-year-cron CronJob.
+# it should be deleted once we have argoCD syncing the cronjob code successfully
+
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: update-current-year-cron
+  namespace: acd38d-dev
+  uid: 3de28c56-9c37-4f0e-8af0-f53d6b8fb1c6
```

Comment:
uid, resourceVersion, generation, createTimestamp, should not be here, they automatically get created.